system-status.sh

```diff
@@ -39,6 +39,9 @@ log minikube_ssh journalctl --boot
 log df -hT
 log minikube_ssh df -hT
 
+# get the rbd-nbd logs from the VM and write them to stdout
+log minikube_ssh "sudo find /var/log/ceph/ -type f | xargs sudo tail -n +1"
```

Comment:
Does `xargs sudo` work for you? I would expect that it needs to be `sudo xargs` instead. It might work, but I would like a confirmation that you manually executed the command or this script.

Alternatively, you can try `"sudo sh -c 'find /var/log/ceph/ -type f | xargs tail -n +1'"`

Reply:
Yes @nixpanic I had manually run it before updating the patch:
```
[pkalever@localhost logs]$ sudo find /tmp/logs/ -type f | xargs sudo tail -n +1
==> /tmp/logs/log2 <==
I might have failed!

==> /tmp/logs/log1 <==
I might have succeeded!
```


Xargs just receives the commands from std Input, so I don't expect it to require special privileges.
As tail is reading the files, it might require.

On the other hand I'm also good with `"sudo sh -c 'find /var/log/ceph/ -type f | xargs tail -n +1'"`

Let me know if you still think your suggestion looks better :-) I can update.

Thanks!

Reply:
If you tested this on minikube over ssh like the script does, than thats fine. Sometimes `sudo` isn't allowed when using `xargs` or pipes, hence my question.

Reply:
Got you! To confirm, yes I have tested it like:
```
[pkalever@localhost ~]$  minikube ssh "sudo find /var/log/ceph -type f | xargs sudo tail -n +1"
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: got: [1100000005000000 WRITE a91000~20000 0] 
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: [1100000005000000 WRITE a91000~20000 0]: finish            
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: waiting for io request                       
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: got: [1200000003000000 WRITE ab1000~20000 0] 
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: [1200000003000000 WRITE ab1000~20000 0]: finish            
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: waiting for io request                       
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: got: [1300000003000000 WRITE ad1000~20000 0] 
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: [1300000003000000 WRITE ad1000~20000 0]: finish            
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: waiting for io request                       
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: got: [1400000003000000 WRITE af1000~20000 0] 
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: [1400000003000000 WRITE af1000~20000 0]: finish            
2021-08-16T10:00:43.439+0000 7f9f69ffb700 20 rbd-nbd: writer_entry: waiting for io request                       
 ```
 
 Thanks!